### ISSUE TITLE
Create portfolio files for all group members.

### ISSUE THREAD

**Issue by yunpeng1234** [AY2122S1-CS2103T-F12-2/tp#218]
Please tick off your name once your porfolio file thing is merged into master.

- [X] Chuan Hao
- [x] Guguz
- [x] Jerome
- [x] Nicole
- [x] Yunpeng

**Comment by yunpeng1234** [comment]
Closed as it is all done
